pkg/operator/operator.go

```diff
@@ -172,3 +176,22 @@ func (o *Operator) Start(ctx context.Context) {
 	}
 	wg.Wait()
 }
+
+func knativeChecker(path string) healthz.Checker {
+	return func(req *http.Request) (err error) {
+		defer func() {
+			if err != nil {
+				fmt.Printf("probe failed: %s", err.Error())
```

Comment:
This isn't going to print in a parsable way for people that are using log parsing. Can we use logging.FromContext(context.Background()) or something similar to get this at least formatted in a way that's readable.